examples/03_connectivity/plot_seed_based_connectome.py

```diff
@@ -60,31 +92,84 @@
     adhd.func[0], confounds=adhd.confounds[0])
 
 ###############################################################################
-# Extract and plot correlation matrix
+# Correlation matrices estimation
+# -------------------------------
+# All starts with estimation of the signals **covariance**. Here the number of
+# samples
+print(power_timeseries.shape[0])
+
+###############################################################################
+# is less than the number of ROIs for Power atlas
+# (and of the same order for Dosenbach atlas). In these situations,
+# graphical lasso **sparse inverse covariance** estimator captures
+# well the covariance **structure**.
+from sklearn.covariance import GraphLassoCV
+
+###############################################################################
+# This estimator can be encompassed into the connectivities estimator object
+# :class:`nilearn.connectome.ConnectivityMeasure`. We set the connectivity
+# *kind* to correlation, to normalize covariance coefficients. 
+
+from nilearn import connectome
+
+correlation_estimator = connectome.ConnectivityMeasure(
+    cov_estimator=GraphLassoCV(verbose=True), kind='correlation')
```

Comment:
Ok. Just for our curiosity here is the difference between GraphLasso and LedoitWolf. What do you prefer ?
<img src="https://cloud.githubusercontent.com/assets/7080143/17615073/d084377e-606b-11e6-9659-f87031985b80.png" width="200"  /> or <img src="https://cloud.githubusercontent.com/assets/7080143/17615077/da8a884a-606b-11e6-867e-5cb8c2d66dd7.png" width="200"  />


If you still think we should use LedoitWolf, then I will do it